Source/KSCrash/Recording/KSCrash.m

```diff
@@ -117,6 +117,59 @@ @implementation KSCrash
 #pragma mark - Lifecycle -
 // ============================================================================
 
++ (void)load
+{
+    [self classDidBecomeLoaded];
```

Comment:
It's better to use `[self class]` here because KSCrash class contains instance methods

Reply:
Fixed